docs/Mutators.md

```diff
@@ -70,6 +71,15 @@ Stryker supports a variety of mutators, which are listed below. Do you have a su
 |`^= ` | `\|= `|
 |`^= ` | `&= ` |
 
+## Collection initialization
```

Comment:
Could probably add a note that this works for everything that uses object initializer syntax, maybe with a link to:https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/object-and-collection-initializers

Reply:
But it doesn't. The whole point (for now) is to only mutate collections. I check if the kind is "Colleciton initializer" and not "Object initializer". We could add the object initializer as well but I'd do that in a seperate PR.

Reply:
In that case ObjectCreationMutator can use a comment to explain that difference as I hadn't caught it from the kind name alone